pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -320,129 +320,129 @@
 
 <!-- Insert Link Dialog -->
 
-<!ENTITY pbInsertLink.label "Enter the Internet address (URL) for this link.">
+<!ENTITY pbInsertLink.label "Digite o endereço de Internet (URL) para este link.">
 <!ENTITY pbInsertLink.accesskey "E">
 
-<!ENTITY pbInsertImage.label "Enter a Internet address (URL) or local file location:">
+<!ENTITY pbInsertImage.label "Digite um endereço de Internet (URL) ou o caminho de um arquivo local:">
 <!ENTITY pbInsertImage.accesskey "E">
 
-<!ENTITY insertListButton.tooltip "Apply or remove bulleted or numbered list">
+<!ENTITY insertListButton.tooltip "Aplicar ou remover lista numerada ou com marcadores">
 
 
 <!--
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY sendAndArchive.label "Send and Archive">
-<!ENTITY sendLaterAndArchive.label "Send Later and Archive">
+<!ENTITY sendAndArchive.label "Enviar e arquivar">
+<!ENTITY sendLaterAndArchive.label "Enviar depois e arquivar">
 
 
 <!--
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY newResponse.label "New Response...">
-<!ENTITY editResponses.label "Edit Responses...">
-<!ENTITY responseButton.tooltip "Insert a response">
-<!ENTITY responseButton.label "Responses">
+<!ENTITY newResponse.label "Nova resposta...">
+<!ENTITY editResponses.label "Editar resposta...">
+<!ENTITY responseButton.tooltip "Inserir uma resposta">
+<!ENTITY responseButton.label "Respostas">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY pbTopicContextMenu-assignColor.accesskey   "c">
-<!ENTITY pbTopicContextMenu-delete.label       "Delete">
+<!ENTITY pbTopicContextMenu-delete.label       "Excluir">
 <!ENTITY pbTopicContextMenu-delete.accesskey   "d">
-<!ENTITY pbTopicContextMenu-assignColor.label  "Set Topic Color">
+<!ENTITY pbTopicContextMenu-assignColor.label  "Definir a cor do tópico">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbStatsWords.label "words:">
+<!ENTITY pbStatsWords.label "palavras:">
 <!ENTITY pbSourceView.accesskey "d">
 <!ENTITY pbSourceViewThemes.accesskey "T">
-<!ENTITY pbStatsChars.label "characters:">
-<!ENTITY formattingToolbar.label "Toolbar">
-<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Goal Status Bar">
-<!ENTITY pbStatsTime.label "time:">
-<!ENTITY pbSourceViewThemes.label "Theme">
-<!ENTITY compositionGoals.tooltip "Show or hide the Composition Goals status bar">
-<!ENTITY switchBetweenNormalAndCode.tooltip "Switch between Normal and Code views">
-<!ENTITY compositionGoals.label "Goals">
+<!ENTITY pbStatsChars.label "caracteres:">
+<!ENTITY formattingToolbar.label "Barra de ferramentas">
+<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Barra de status do objetivo">
+<!ENTITY pbStatsTime.label "horário:">
+<!ENTITY pbSourceViewThemes.label "Tema">
+<!ENTITY compositionGoals.tooltip "Mostrar ou ocultar a barra de status de Objetivos de Composição">
+<!ENTITY switchBetweenNormalAndCode.tooltip "Alternar entre modos Normal e Código">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY switchBetweenNormalAndCode.tooltip "Alternar entre modo Normal e de Código">